CIRG-PainTracker-STOP.json

```diff
@@ -2,7 +2,7 @@
   "resourceType": "Questionnaire",
   "id": "CIRG-PainTracker-STOP",
   "title": "STOP Questionnaire: A Tool to Screen Patients for Obstructive Sleep Apnea",
```

Comment:
@mcjustin this questionnaire is also missing a `name` element, like the others in the repo.